va_explorer/va_data_management/models.py

```diff
@@ -613,19 +613,13 @@ class VerbalAutopsy(models.Model):
     # Automatically set timestamps
     created = models.DateTimeField(auto_now_add=True)
     updated = models.DateTimeField(auto_now=True)
-
-    # function to tell if VA had any coding errors
+      # function to tell if VA had any coding errors
     def any_errors(self):
         return self.coding_issues.filter(severity='error').exists()
-
-    # function to tell if VA had any coding warnings
+        # function to tell if VA had any coding errors
```

Comment:
The diff suggests the formatting is off here in a weird way for some reason. Perhaps a running the formatter on this file would quickly correct?

Reply:
Fixed